vars/sectionBuildAndTest.groovy

```diff
@@ -47,7 +47,7 @@ def call(params) {
     }
   }
 
-  stage("Tests/Checks/Container build") {
+  stage("Tests/Checks") {
```

Comment:
is it possible to somehow extend the tests/checks step to run multiple different tests in parallel?

Reply:
Maybe. What tests do you have in mind?

Reply:
Well typically we have unit tests and integration tests. They are independent and can run in parallel. No doubt there can be more at this stage

Reply:
Integration tests might need a deployment. However loosing the parallelisation make some project build longer, it was the whole point of this parallelisation. e.g. (I count the shortest)

div-decree-nisi-frontend 1:35 min
probate-frontend 3:30 min
probate-caveat-frontend 1:20 min
div-case-orchestration-service 3:00 min
div-petitioner-frontend 2:20 min

Reply:
perhaps up to a project. normally integration tests don't need deployment. functional tests do. i was more on topic whether individual project incorporating this jenkins library can customly extend this stage/step by including additional parallel functions

Reply:
Oh sure ok. But as is, these steps can't be extended easily... not sure this even possible.